src/engine/V3Eval.v3

```diff
@@ -588,6 +602,54 @@ def I8_POPCNT(a: u8) -> u8 {
 	}
 	return count;
 }
+def I8_ADD_SAT_S(a: i8, b: i8) -> i8 {
+	var sum = i16.view(a) + i16.view(b);
+	if (sum > 127) return 127;
+	else if (sum < -128) return -128;
+	else return i8.view(sum);
+}
+def I8_ADD_SAT_U(a: u8, b: u8) -> u8 {
+	var sum = u16.view(a) + u16.view(b);
+	if (sum > 255) return 255;
```

Comment:
Can you also do the `return if(` here and elsewhere in this file?

Reply:
Yes just found out I should do that after the new commit. Pushing a new one soon.